plugins/module_utils/network/ios/config/snmp_server/snmp_server.py

```diff
@@ -34,6 +34,20 @@
 )
 
 
+def handle_deprecates(wantd):
```

Comment:
Amazing work! Could you please encapsulate this function within a class and make it accessible through 'self'? Additionally, there would be a merge conflict stemming from our recent merge of your pull request #900 . Could you kindly resolve that as well?

Reply:
@earendilfr CC ^ we would like to get this one merged, if you may consider the review.

Reply:
Hello,
Sorry, I don't have seen the notification concerning your remark.
Merge done with the remarks